crates/common/src/io.rs

```diff
@@ -12,6 +12,9 @@ cfg_if! {
     } else if #[cfg(target_arch = "riscv64")] {
         #[doc = "Concrete implementation of the [BasicKernelInterface] trait for the `riscv64` target architecture."]
         pub type ClientIO = crate::asterisc::io::AsteriscIO;
+    } else if #[cfg(target_os = "zkvm")] {
+        #[doc = "Concrete implementation of the [BasicKernelInterface] trait for the `zkvm` target architecture."]
+        pub type ClientIO = crate::zkvm::io::ZkvmIO;
```

Comment:
I would like to get rid of the need for any of these changes, but putting them in here for now so it compiles and works.

Reply:
Totally right. Clabby and I brainstormed a few alternatives to remove the need for it, since we don't actually use the io system so it's not doing anything.

He's thinking through it more, but my perspective is that best option would be to create a no op BKI implementation and organize the feature flags and defaults in such a way that it typically defaults to NativeIO, but that we can import with default features off to fall back to the no op.